msmbuilder/commands/example_datasets.py

```diff
@@ -57,3 +57,9 @@ class MetEnkephalinDatasetCommand(DatasetCommand):
     _concrete = True
     klass = MetEnkephalin
     description = 'Download example Met-Enkephalin dataset.'
+
+
+class MinimalFsPeptideDatasetCommand(DatasetCommand):
```

Comment:
I don't think this will work (meaningfully) from the command line. It uses the same data as FsPeptide but loads with a stride parameter. The command line commands just dump the data, so it would be the same as FsPeptide